public/app/features/explore/Logs/Logs.tsx

```diff
@@ -265,38 +265,37 @@ const UnthemedLogs: React.FunctionComponent<Props> = (props: Props) => {
     }
 
     // check if we have dataFrames that return the same level
-    const logLevelsArray: Array<{ levelStr: string; logLevel: LogLevel }> = [];
```

Comment:
No significant changes here, mainly a clean up of unnecessary complexity. It's enough to deal with an array of LogLevel[], no need for the string version.